static/js/home.js

```diff
@@ -149,3 +167,39 @@ $(document).ready( function(){
     })
   })
 })
+
+//.......Event to upload to particular directory..................
+ 
+//new_parent = ''
+//
+ //});
+
+$(document).on('click','#upload-file-btn',function(e){
+  var n = new_parent
+  //alert(n)
+ var form_data = new FormData($('#upload-form')[0])
+        if(n[0] == '/')
+        n = new_parent.slice(1,(n.length))
+        form_data.append('path',n)
+          console.log('Starting upload')
+          $.ajax({
+              type: 'POST',
+              url: '/upload',
+              data: form_data,
+              contentType: false,
+              processData: false,
+              success: function(response) {
+                console.log(response)
+              },
+
+              error: function() {
+                console.log('Error uploading file.')
+              }
+          });
+      });
+ 
+
```

Comment:
Leave exactly one blank line at the end of the file. Don't leave more than that.